include/gz/sim/components/Environment.hh

```diff
@@ -45,20 +45,33 @@ namespace components
     using FrameT = common::DataFrame<std::string, T>;
     using ReferenceT = math::SphericalCoordinates::CoordinateType;
 
+    /// \brief Reference units
+    enum ReferenceUnits {
+      RADIANS = 0,
+      DEGREES
+    };
```

Comment:
@arjo129 meta: should we add a `METERS` entry so that the units fields makes sense for other coordinate systems?

Reply:
I'm not sure at the moment if this is nessecary.